specs/content-templates.spec.ts

```diff
@@ -31,7 +31,7 @@ test('segmented content template works', async ({ page }) => {
     <p>Want to see more?</p>
 </div>
 </div>
-<div class="os-raise-ib-cta" data-button-text="Yes!" data-fire-event="event2" data-wait-for-event="event1">
+<div class="os-raise-ib-cta" data-button-text="Yes!!" data-fire-event="event2" data-wait-for-event="event1">
```

Comment:
Previously, we were removing the button, so when the test would look for the `Yes!` button the second time there was only one button showing `Yes!`. Now that we are instead disabling the button, there were two `Yes!` buttons and the test was failing because there were two `Yes!` buttons that could be clicked.

Reply:
Ah, right. Thanks!